frontend-public/src/tests/HOC/__snapshots__/authenticationGuard.spec.js.snap

```diff
@@ -1,64 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`AuthenticationGuard should render the \`WrappedComponent\` if \`isAuthenticated\` 1`] = `
-ShallowWrapper {
-  Symbol(enzyme.__root__): [Circular],
-  Symbol(enzyme.__unrendered__): <authenticationGuard
-    getUserInfoFromToken={[MockFunction]}
-    isAuthenticated={true}
-  />,
-  Symbol(enzyme.__renderer__): Object {
-    "batchedUpdates": [Function],
-    "getNode": [Function],
-    "render": [Function],
-    "simulateError": [Function],
-    "simulateEvent": [Function],
-    "unmount": [Function],
-  },
-  Symbol(enzyme.__node__): Object {
-    "instance": null,
-    "key": undefined,
-    "nodeType": "function",
-    "props": Object {
-      "getUserInfoFromToken": [MockFunction],
-      "isAuthenticated": true,
-    },
-    "ref": null,
-    "rendered": null,
-    "type": [Function],
-  },
-  Symbol(enzyme.__nodes__): Array [
-    Object {
-      "instance": null,
-      "key": undefined,
-      "nodeType": "function",
-      "props": Object {
-        "getUserInfoFromToken": [MockFunction],
-        "isAuthenticated": true,
-      },
-      "ref": null,
-      "rendered": null,
-      "type": [Function],
-    },
-  ],
-  Symbol(enzyme.__options__): Object {
-    "adapter": ReactSixteenAdapter {
-      "options": Object {
-        "enableComponentDidUpdateOnSetState": true,
-        "lifecycles": Object {
-          "componentDidUpdate": Object {
-            "onSetState": true,
-          },
-          "getDerivedStateFromProps": true,
-          "getSnapshotBeforeUpdate": true,
-          "setState": Object {
-            "skipsComponentDidUpdateOnNullish": true,
-          },
-        },
-      },
-    },
-    "attachTo": undefined,
-    "hydrateIn": undefined,
-  },
-}
+<Component
+  getUserInfoFromToken={[MockFunction]}
+  isAuthenticated={true}
+/>
```

Comment:
I'm concerned by the lack of content in this snap. It has almost no details about the state of the component

Reply:
Were we using snapshots to describe the state of the component? I thought the intent was to ensure that any potential UI changes were identified. In that sense, wouldn't state be an implementation detail tested through unit and integration tests? Also, for this specific instance - isn't the state just the internals of the Enzyme renderer, rather than anything to do with the component?

I was tracing through the snapshots because of this [Jest issue](https://github.com/facebook/jest/issues/7802), wherein someone raises a similar issue about snapshotting the private methods of the Enzyme library. One of the jest contributors then suggests needing a proper serializer to flatten the component structure, ala enzyme-to-json. Though I'm not sure of the configurations of the enzyme-to-json, it may be able to provide us a bit more information.

Reply:
Apologies, I shouldn't have used the term "state" as what I really meant was the values of all values in the component based on the props that it received. We provide props when generating the snapshots.

Regardless, I think you're right anyway. I didn't look carefully enough at 1) Which component snap I was comparing -- authenticationGuard has very little content and 2) What the majority of the diff removal was showing.

It looks like the props are being properly passed into the snapshots. I'll mark this as resolved